src/main/java/core/basesyntax/figure/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.figure;
+
+public class Square extends AbstractFigure implements Figure {
+    private double side;
+
+    public Square(String color, double side) {
```

Comment:
Don't pass expressions in the constructor. It's better to create a variable for the 'side' parameter and pass it to the constructor for better code readability.